docker/conf/docker-compose.demo.yml.dist

```diff
@@ -0,0 +1,61 @@
+version: "3.4"
+services:
+    postgres:
+        image: postgres:10.5-alpine
+        volumes:
+            - ./docker/postgres/postgres.conf:/var/lib/postgresql/data/postgresql.conf
+        environment:
+            - PGDATA=/var/lib/postgresql/data/pgdata
+            - POSTGRES_USER=root
+            - POSTGRES_PASSWORD=root
+            - POSTGRES_DB=shopsys
+
+    webserver:
+        image: nginx:1.13-alpine
+        depends_on:
+            - php-fpm
+        volumes:
+            - web-volume:/var/www/html/web
```

Comment:
OK, persisting of content directory was not part of MVP.
But web-volume will be created as `$DEMO_NAME_web-volume` after some update of project there is need to recreate web-volume because it will be filled with old data. 
Also in the case of `wipe_workspace.sh` it is important to remove web-volume but `docker-compose down -v` should handle named volumes https://docs.docker.com/compose/reference/down/. please update wipe_workspace with that if it is used for demoshops